src/scaffold/entry_type/fields.rs

```diff
@@ -45,25 +46,99 @@ pub fn choose_fields(
     }
 
     println!(
-        "Chosen fields:\n {}",
+        "Current fields:\n{}\n",
         fields
             .iter()
-            .map(|f| format!("{}: {}", f.field_name.clone(), f.field_type))
+            .map(|f| format!(" {}: {}", f.field_name.clone(), f.field_type))
             .collect::<Vec<String>>()
-            .join(", ")
+            .join("\n")
             .italic()
     );
 
     let selection = Select::with_theme(&ColorfulTheme::default())
-        .with_prompt(
-            "Do you want to proceed with the current entry type or restart from the beginning?",
-        )
-        .item("Confirm")
-        .item("Restart")
+        .with_prompt("Do you want to proceed with the current entry type?")
+        .items(&["Confirm", "Modify", "Restart"])
         .default(0)
         .interact()?;
 
     if selection == 1 {
+        loop {
+            let action = Select::with_theme(&ColorfulTheme::default())
+                .with_prompt("What would you like to do?")
+                .items(&["Change Field", "Add Field", "Remove Field", "Done"])
+                .interact()?;
+
+            match action {
+                0 => {
+                    // Change field
+                    if !fields.is_empty() {
+                        let field_to_change = Select::with_theme(&ColorfulTheme::default())
+                            .with_prompt("Select field to change")
+                            .items(
+                                &fields
+                                    .iter()
+                                    .map(|f| format!("{}: {}", f.field_name, f.field_type).italic())
```

Comment:
```suggestion
                                    .map(|f| format!("{}: {}", f.field_name, f.field_type))
```

The formatting on these menu options made me think it was informational text rather than things I could select, given that it was the only italic menu and italic had previously been used as info text. It took me a while to notice the menu!

I realise I'm giving this input on something that's only a backport PR though, so I could do this as a separate PR against 0.5 and 0.4.

Reply:
Ahh wasn't aware of this confusion, can look into this, thanks